integration-tests/azure-storage-blob/src/main/java/io/quarkiverse/azure/storage/blob/it/StorageBlobAsyncResource.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.quarkiverse.azure.storage.blob.it;
+
+import static jakarta.ws.rs.core.Response.Status.CREATED;
+import static jakarta.ws.rs.core.Response.Status.NOT_FOUND;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.*;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+
+import com.azure.core.util.BinaryData;
+import com.azure.storage.blob.BlobAsyncClient;
+import com.azure.storage.blob.BlobServiceAsyncClient;
+import com.azure.storage.blob.models.BlobItem;
+import com.azure.storage.blob.models.BlockBlobItem;
+
+import io.smallrye.mutiny.Multi;
+import io.smallrye.mutiny.Uni;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+@Path("/quarkus-azure-storage-blob-async")
+@Produces(MediaType.TEXT_PLAIN)
+@ApplicationScoped
+public class StorageBlobAsyncResource {
+
+    @Inject
+    BlobServiceAsyncClient blobServiceAsyncClient;
+
+    @Path("/{container}/{blobName}")
```

Comment:
Do we need add some java doc?

Reply:
This is integration test sample, java doc is nice to have. When user uses our extension, the exposed/produced `BlobServiceAsyncClient` from azure java sdk is used in user's code, from which user can see any java doc associated.